libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionIdentityType.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.di.ipv.core.library.auditing.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.enums.IdentityType;
+import uk.gov.di.ipv.core.library.enums.Vot;
+
+@ExcludeFromGeneratedCoverageReport
+public record AuditExtensionIdentityType(
+        @JsonProperty(value = "identity_type", required = true) IdentityType identityType,
+        @JsonProperty(required = false) Vot vot)
```

Comment:
I don't think the `required` will do anything here (but it doesn't do any harm 🤷)

Reply:
Yeh I just thought it would make it more obvious that one is required and one not :)